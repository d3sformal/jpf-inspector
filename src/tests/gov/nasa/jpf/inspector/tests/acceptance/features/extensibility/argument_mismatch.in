hi          too many arguments
arg1        still too many arguments
less_than   not_enough_arguments