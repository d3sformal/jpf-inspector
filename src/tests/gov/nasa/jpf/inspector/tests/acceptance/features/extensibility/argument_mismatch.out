ERR: You have provided 3 arguments which is more than what the alias 'hi' requires (0). This may not be what you intended.
The alias should expand to this: 'hello'.
ERR: You have provided 4 arguments which is more than what the alias 'arg1' requires (1). This may not be what you intended.
The alias should expand to this: 'create breakpoint pos=*:{0}'.
ERR: The alias 'less_than' requires 3 arguments but only 1 were provided.
The alias should expand to this: 'assert *:{0} {1} < {2}'.